lib/Service/Sound.cpp

```diff
@@ -93,34 +93,35 @@ static const uint8_t VOLUME = 10;
 
 void Sound::playAlarm()
 {
-    IBuzzer& buzzer = Board::getInstance().getBuzzer();
-
-    /* Req. 3.4.5-2:

```

Comment:
These changes are dangerous later for the merge. It should not be necessary in case you removed the calls in your app.

Reply:
I removed the Buzzer from the Board to save Flash space. Therefore, I can't access the buzzer over Board::getInstance().getBuzzer();

I added a TODO to avoid forgetting about that issue.

Reply:
Ok.